libs/sdk-react-native/src/index.tsx

```diff
@@ -559,9 +598,8 @@ export const sendSpontaneousPayment = async (nodeId: string, amountSats: number)
     return response as Payment
 }
 
-export const receivePayment = async (amountSats: number, description: string): Promise<LnInvoice> => {
-    const response = await BreezSDK.receivePayment(amountSats, description)
-    return response as LnInvoice
+export const receivePayment = async (reqData: ReceivePaymentRequest): Promise<ReceivePaymentResponse> => {
+    return await BreezSDK.receivePayment(reqData)
```

Comment:
The preimage Uint8Array needs to be converted to a number[] to be passed over the RN bridge
```
    return await BreezSDK.receivePayment({
        ...reqData,
        preimage: reqData.preimage && Array.from(reqData.preimage)
    })
```

Reply:
Done as part of 2b40b13c05d6a84425019c0c35d88d7289705e3c